java-client/src/main/java/org/opensearch/client/transport/aws/AwsSdk2Transport.java

```diff
@@ -10,10 +10,7 @@
 
 import jakarta.json.JsonObject;
 import jakarta.json.stream.JsonParser;
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.UnsupportedEncodingException;
+import java.io.*;
```

Comment:
Could you please revert to non-`*` import? (`spotless` should have spotted that)

Reply:
Done. I think `spotless` actually made this change. My IDE is set to never do wildcards. Either way, it's fixed.